lib/lutaml/model/xml_adapter/builder/nokogiri.rb

```diff
@@ -39,13 +39,15 @@ def create_and_add_element(
           )
             add_namespace_prefix(prefix)
 
+            tag_name = prefix ? "#{prefix}:#{element_name}" : element_name
```

Comment:
@Novtopro Currently the `add_namespace_prefix(prefix)` method is being used to add prefix to the tag name, adding it again here will cause the namespace to be duplicated as we can see in the specs.

Can you provide an example XML that is causing this issue? Thanks

Reply:
Yeah, my fix does not work. Anyway, I left a comment in https://github.com/metanorma/niso-jats/pull/19#issuecomment-2449677983, could you please take a look?

Reply:
Sure, I'll look into it.